### PR TITLE
User slugs

### PR DESCRIPTION
**Pull request by matthewdias**
👨  🏷

### CODE REVIEW

**Comment by matthewdias** [comment]
Just need to fix the build on this one and it should be good to go.

File: app/routes/users.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default Route.extend(DataErrorMixin, CanonicalRedirectMixin, CoverPageMix
   },
 
   serialize(model) {
-    return { name: get(model, 'name') };
+    return { slug: get(model, 'slug') };
```
Comment (NuckChorris) [inline app/routes/users.js:35]: Don't we also need to have `|| get(model, 'id')` to have it fallback to numerical ID in link generation?

Comment (trmcnvn) [reply app/routes/users.js:35]: Yeah we should, but I don't think there is anywhere where we create a link without knowing the slug.

Comment (NuckChorris) [reply app/routes/users.js:35]: slug can be missing if a user hasn't chosen one yet, so we probably need to handle that

Comment (trmcnvn) [reply app/routes/users.js:35]: Really? Figured it was auto-set to their name since mine was on production API.

Comment (NuckChorris) [reply app/routes/users.js:35]: That was just until we release the ability to pick slugs, it's not a permanent thing.

**Pull request merged**